# CUPS_PDFTOPS should resolve to poppler binary
# and not the binary from cups-filters. Upstream bug:
# https://bugs.linuxfoundation.org/show_bug.cgi?id=1417
# Fixed upstream in 1.17.10:
# https://github.com/OpenPrinting/cups-filters/commit/b72c0bf80f21085521508900130ec553fae089b7

diff --git a/configure.ac b/configure.ac
index df7fed5..1c8727e 100644
--- a/configure.ac
+++ b/configure.ac
@@ -668,7 +668,7 @@ AS_IF([test "x$with_pdftops_path" != "xsystem"], [
 	CUPS_PDFTOPS="$with_pdftops_path"
 ], [
 	AS_IF([test "x$cross_compiling" = "xyes"], [
-		CUPS_PDFTOPS="pdftops"
+		CUPS_PDFTOPS="/usr/bin/pdftops"
 	], [
 		AC_CHECK_PROG(CUPS_PDFTOPS, pdftops, /usr/bin/pdftops)
 	])
